### PR TITLE
Preference in Organization list

### PR DESCRIPTION
**Pull request by abdussamadbsg**
Preference linked to the Organization list

### CODE REVIEW

File: lib/ui/view/organization/organization_view/organization_view.dart [hunk 1]
```diff
@@ -53,10 +53,10 @@ class OrganizationView extends StatelessWidget {
                                     itemCount: model.organizations.length,
                                     shrinkWrap: true,
                                     itemBuilder: (context, i) {
-                                     final org = model.organizations[i];
-                                     print (org.id);
-                                     print (org.organizationUrl);
-                                     print (org.name);
+                                      final org = model.organizations[i];
```
Comment (Blazebrain) [inline lib/ui/view/organization/organization_view/organization_view.dart:56]: Remove the priint statements in this screen

Replace them with the log.i()

**Review by Blazebrain — CHANGES_REQUESTED** [review]
Good to go

Asides that little tweak

Fix the change and we good to go

**Comment by YenumJay** [comment]
Pull latest changes from dev and resolve conflicts.

**Review by jeremiahseun — DISMISSED** [review]
LGTM

File: lib/ui/view/organization/organization_view/organization_view.dart [hunk 2]
```diff
@@ -54,9 +54,9 @@ class OrganizationView extends StatelessWidget {
                                     shrinkWrap: true,
                                     itemBuilder: (context, i) {
                                       final org = model.organizations[i];
-                                      print(org.id);
-                                      print(org.organizationUrl);
-                                      print(org.name);
+                                      model.log.i(org.id);
+                                      model.log.i(org.organizationUrl);
+                                      model.log.i(org.name);
```
Comment (pauleke65) [inline lib/ui/view/organization/organization_view/organization_view.dart:59]: Move all Log and logic statements to the view model

File: lib/ui/view/organization/organization_view/organization_viewmodel.dart [hunk 3]
```diff
@@ -119,4 +119,8 @@ class OrganizationViewModel extends BaseViewModel {
 
   String? get currentOrgId =>
       storageService.getString(StorageKeys.currentOrgId);
-}
\ No newline at end of file
+
+  Future viewPreferences() async {
```
Comment (pauleke65) [inline lib/ui/view/organization/organization_view/organization_viewmodel.dart:123]: change Future to Future<void>

**Review by pauleke65 — CHANGES_REQUESTED** [review]
Changes Requested

**Review by pauleke65 — DISMISSED** [review]
Terrific👍🏽

**Comment by jeremiahseun** [comment]
Please fix conflicts

**Review by pauleke65 — APPROVED** [review]
LG

**Review by YenumJay — APPROVED** [review]
Good to go

**Pull request merged**